src/Provider/3dTilesProvider.js

```diff
@@ -7,6 +7,83 @@ import Extent from 'Core/Geographic/Extent';
 import { pre3dTilesUpdate, process3dTilesNode, init3dTilesLayer } from 'Process/3dTilesProcessing';
 import utf8Decoder from 'Utils/Utf8Decoder';
 
+/**
+ * Class mapping 3D Tiles extensions names to their associated parsing methods
+ */
+export class $3DTilesExtensions {
```

Comment:
Do we really want to export it if we export its instance below ?

Reply:
Yep you're right, I removed the export.